docusaurus/src/connector_registry.js

```diff
@@ -54,8 +54,8 @@ function getSupportLevelDisplay(rawSupportLevel) {
 module.exports = {
   REGISTRY_URL,
   catalog: fetchCatalog(),
-  isPypiConnector: (connector) => {
-    return Boolean(connector.remoteRegistries_oss?.pypi?.enabled);
+  isPyAirbyteConnector: (connector) => {
+    return Boolean(connector.language_oss == "manifest-only" || connector.remoteRegistries_oss?.pypi?.enabled);
```

Comment:
@bnchrch I tried, but this did not do the trick. What am I missing here? Perhaps I misunderstood what's in this registry entry key — I will verify now.